ops/__init__.py

```diff
@@ -12,31 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""The ops library: a Python framework for writing Juju charms.
+"""The API to respond to Juju events and manage the application.
 
-The ops library is a Python framework (`available on PyPI`_) for developing
```

Comment:
Did you drop the link to PyPI because you feel like that's where people expect to get most Python packages, so there's no point? Or because you think it doesn't belong in reference docs? Some other reason? I'm not sure I'm totally onboard, so reasoning would help convince me.